orangecontrib/text/tests/test_bag_of_words.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from unittest.mock import patch
+
+from orangecontrib.text.bagofowords import BagOfWords
+from orangecontrib.text.corpus import Corpus
+
+
+def progress_callback():
+    return
+
+
+def error_callback(*args):
+    return
+
+
+def mock_gensim_dictionary():
+    raise Exception
+
+
+class BagOfWordsTests(unittest.TestCase):
+
+    def setUp(self):
+        self.bow = BagOfWords(
+                progress_callback=progress_callback,
```

Comment:
Is there a reason why this cannot be just `self.bow = BagOfWords()`? Then we could also delete the `progress_callback` and `error_callback` methods above.